app/main.js

```diff
@@ -94,6 +96,22 @@ app.on('before-quit', () => {
 })
 
 async function onready () {
+  const { app, Menu, Tray } = require('electron')
+
+  let appIcon = null
+  app.whenReady().then(() => {
+    appIcon = new Tray(LOGO_FILE)
+    const contextMenu = Menu.buildFromTemplate([
+      { label: 'Show/Hide', type: 'radio' },
+      { label: 'Quit', type: 'radio' }
```

Comment:
We should have the quit button actually quit

```suggestion
      {
        label: 'Quit',
        role: 'quit'
      }
```